doc/classes/TreeItem.xml

```diff
@@ -96,6 +96,14 @@
 				Returns the button index if there is a button with ID [param id] in column [param column], otherwise returns -1.
 			</description>
 		</method>
+		<method name="get_button_color" qualifiers="const">
+			<return type="Color" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="id" type="int" />
+			<description>
+				Returns the color of the button with ID [param id] in column [param column]. If the specified button does not exists, returns [code skip-lint]Color()[/code].
```

Comment:
> Currently, there's no way to link to constuctors with BBCode style tags. I suppose I could have this method return Color.TRANSPARENT, because you are able to reference constants in the docs

There's no much point to reference constructors in the class reference, yet. Even for this example, you can replace it and it would be more understandable in my opinion. "Clearer", if you will.
```suggestion
				Returns the color of the button with ID [param id] in column [param column]. If the specified button does not exists, returns [constant Color.TRANSPARENT].
```